models/packages/descriptor.go

```diff
@@ -99,7 +101,11 @@ func GetPackageDescriptor(ctx context.Context, pv *PackageVersion) (*PackageDesc
 	}
 	creator, err := user_model.GetUserByID(ctx, pv.CreatorID)
```

Comment:
No, if the user gets deleted, the `CreatorID` is not -1 or -2.

Reply:
So maybe we need to check if it's -2 here?

Reply:
Does actions user have write permission to create packages?

Reply:
> Does actions user have write permission to create packages?

I think it doesn't. Gitea will use the trigger user instead of actions bot user in #23729.

However, I think it will be more robust to support actions bot as package creator. (BTW, not the owner)

Reply:
Yes, but it is not merged. It seems that actions user can not create packages in current version, but I’m not sure about this, so I asked this question.

Reply:
> Yes, but it is not merged. It seems that actions user can not create packages in current version, but I’m not sure about this, so I asked this question.

OK, I misunderstand your question. So the answer to `Does actions user have write permission to create packages` is no.